core/src/main/java/org/kiwix/kiwixmobile/core/search/viewmodel/SearchViewModel.kt

```diff
@@ -127,8 +126,7 @@ class SearchViewModel @Inject constructor(
       searchResultsFromZimReader(),
       filter,
       Function3(this::reduce)
-    ).debounce(100, MILLISECONDS)
```

Comment:
Why was this added? :/

Reply:
I think in an earlier incarnation of the screen I didn't have an appropriate `Initialising` state so there was a brief flicker as the screen moved states on creation. All in all a poor decision by myself that is now being corrected